Tutorials/whatiswot/10-Application_Domains/10-Application-Domains.vtt

```diff
@@ -0,0 +1,119 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:06.925 
+In this video, we will give a introduction to the W3C Web of Things including its
+
+00:00:06.925 --> 00:00:13.850 
+application domains and we will continue with architecture patterns, and building blocks in the upcoming videos.
+
+00:00:13.850 --> 00:00:17.125 
+So let's start with what is WoT.
+
+00:00:17.125 --> 00:00:24.200 
+It is a set of standards defined for the interoperability of different Internet
+
+00:00:24.200 --> 00:00:31.275 
+of Things platforms and application domains by the World Wide Web Consortium (W3C).
```

Comment:
too much pause before World.

Also, I think it is better to move "World Wide Web Consortium" to the sentence above, right after "defined" otherwise it might be understood as if the domains are from w3c